### PR TITLE
Remove Success Page from Crypto Flow

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4965

crypto-flow-fix

### CODE REVIEW

**Review by znarf — COMMENTED** [review]
@BenJam needs to confirm first he's fine with the behavior.

**Comment by BenJam** [comment]
left a comment on the original issue over at https://github.com/opencollective/opencollective/issues/4935#issuecomment-979070288

**Comment by SudharakaP** [comment]
> left a comment on the original issue over at [opencollective/opencollective#4935 (comment)](https://github.com/opencollective/opencollective/issues/4935#issuecomment-979070288)

Thanks. 👍🏽 

@znarf @BenJam : I've made changes to this as per the new design. Let me know how it goes. 🙂

File: components/contribution-flow/StepCheckout.js [hunk 1]
```diff
@@ -7,13 +7,57 @@ import { FormattedMessage } from 'react-intl';
...
+        <Flex mt="24px" justifyContent="center">
+          <P fontSize="13px" color="black.700">
+            <FormattedMessage
+              defaultMessage="You can <SupportLink>contact support</SupportLink> if something goes wrong."
+              values={{
+                SupportLink: I18nSupportLink,
+              }}
+            />
+          </P>
+        </Flex>
```
Comment (znarf) [inline components/contribution-flow/StepCheckout.js:56]: I don't see the point for this, the interface is already cluttered by the MessageBox above.

Comment (SudharakaP) [reply components/contribution-flow/StepCheckout.js:56]: This is as per design; https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=15015%3A205718. I think the idea is that if the QR code is not generated for some reason the user has a way of contacting support on the page itself. But I am not against removing it so I'll do so if no objections.

**Comment by BenJam** [comment]
LGTM.

File: components/contribution-flow/StepCheckout.js (hunk 1 above)
Comment (SudharakaP) [reply components/contribution-flow/StepCheckout.js]: I've removed the link. 👍🏽

**Pull request merged**